README.rst

```diff
@@ -0,0 +1,54 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2022 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+==============================
+ Cookiecutter-Invenio-ILS
+==============================
+
+.. image:: https://img.shields.io/github/license/inveniosoftware/cookiecutter-invenio-ils.svg
+        :target: https://github.com/inveniosoftware/cookiecutter-invenio-ils/blob/master/LICENSE
+
+.. image:: https://img.shields.io/travis/inveniosoftware/cookiecutter-invenio-ils.svg
+        :target: https://travis-ci.org/inveniosoftware/cookiecutter-invenio-ils
+
+This `Cookiecutter <https://github.com/audreyr/cookiecutter>`_ template is
+designed to help you to bootstrap an `InvenioILS
+<https://github.com/inveniosoftware/invenio-app-ils>`_ service.
+
+Quickstart
+----------
+
+Install the latest Cookiecutter if you haven't installed it yet::
+
+    pip install -U cookiecutter
+
+Generate your Invenio ILS instance::
+
+    cookiecutter https://github.com/inveniosoftware/cookiecutter-invenio-ils.git -c v1.0
+
+More detailed steps can be found in the https://inveniosoftware.org/#getstarted
```

Comment:
The link is not correct, shall we put <https://invenioils.docs.cern.ch> instead?